test/reporters/tracer.go

```diff
@@ -0,0 +1,134 @@
+package reporters
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/Microsoft/ApplicationInsights-Go/appinsights"
+	"github.com/Microsoft/ApplicationInsights-Go/appinsights/contracts"
+	"github.com/sirupsen/logrus"
+)
+
+// skip log lines which are just artifacts of ginkgo
+var exclude = []string{"", "S\n", "SS\n", "========================\n", "------------------------------\n", "•\n"}
```

Comment:
i feel like this could very well be a list of regexes, the number of "S" depends on how many tests are skipped and that can change, the dot lines will change according to how many tests are executed, and the lines' length depend on the screen size.

Reply:
agreed. Will try to see how we can do this without engineering rocket